src/components/Products.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import ProductsCard from './ProductsCard'
+
+const Products = ({ products }) => {
+    console.log(products)
+    return (
+        <div>
+            <h1>Products</h1>
+            <div className='products-card-box'>
+                {products.map((item) => {
```

Comment:
What are we showing if there are not products? I think that may be the edge case we have here.

Reply:
Is edge case by the array length a normal solution?